### DISCUSSION TITLE
Связанные селекты из нескольких таблиц

### DISCUSSION THREAD

**Discussion by EKhorikov** [totumonline/totum-ru-issues-and-discussions#64] [Как это сделать?] [answered]
Добрый день.
Есть таблица циклов "Заявки", в которой создаются заявки на обслуживание оборудования, через расчётную таблицу в цикле, соответственно.
Так же, есть таблица 2 - "Контрагенты" в которой заводятся карточки контрагентов.
Таблица 3 - "Объекты контрагента" в которой заводятся карточки объектов, привязанные к контрагенту.
Задача: При создании новой заявки, в поле 1 выбирается "Контрагент", в поле 2 ДОЛЖЕН выбираться "Объект контрагента". Т.е. в поле 2 должен открываться селект, в котором только объекты выбранного ранее контрагента.
Как отфильтровать отображение селекта в поле 2, чтоб все другие объекты из таблицы "Объекты контрагента" не отображались?

![image](https://user-images.githubusercontent.com/92679628/151699284-236f0fb3-2233-4635-9254-78b40ae2bbb1.png)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
### [Различный состав Селекта в зависимости от других полей](https://docs.totum.online/different-composition-of-select-depending-on-other-fields)
